src/content/hcp/product-landing.json

```diff
@@ -32,6 +32,11 @@
 				"icon": "boundary-color",
 				"text": "HCP Boundary",
 				"url": "/boundary/tutorials/hcp-getting-started"
+			},
+			{
+				"icon": "waypoint-color",
+				"text": "HCP Waypoint",
+				"url": "/waypoint/tutorials/runners/hcp-runners-on-your-infra"
 			}
```

Comment:
I think maybe we just don't include waypoint here since there isn't a tru getting started collection? I'll defer to @topfunky though.

Reply:
I'd like to leave it in as it highlights that HCP Waypoint is available if someone is browsing for HCP services. 

Unfortunately it's the only tutorial we have so not a true getting started though you're right 😅